src/WindowsTracing/KrabsTracer.h

```diff
@@ -29,20 +30,33 @@ class KrabsTracer {
  private:
   void SetTraceProperties();
   void EnableProviders();
+  void EnableSystemProfilePrivilege(bool value);
+  void SetupStackTracing();
   void Run();
   void OnThreadEvent(const EVENT_RECORD& record, const krabs::trace_context& context);
+  void OnStackWalkEvent(const EVENT_RECORD& record, const krabs::trace_context& context);
   void OutputStats();
 
+  struct Stats {
+    uint64_t num_thread_events = 0;
+    uint64_t num_profile_events = 0;
```

Comment:
Where is this incremented?

Reply:
removed unused num_profile_events, thanks.